config/activator.yaml

```diff
@@ -44,12 +44,14 @@ spec:
         # and substituted here.
         image: knative.dev/serving/cmd/activator
         ports:
-        - name: http1-port
+        - name: http1
           containerPort: 8012
-        - name: h2c-port
+        - name: h2c
           containerPort: 8013
-        - name: metrics-port
```

Comment:
These changes seem unnecessary.

Reply:
Yeah. I just wanted to unify the approach across activator, autoscaler, controller, webhook. Only here in activator we use the `-port` suffix. Everywhere else it's omitted. And I think it's actually redundant because this whole section is under "ports" section. WDYT?

Reply:
I agree with that. Mind splitting this out in a separate PR?

Reply:
OK.